atom/browser/api/atom_api_window.cc

```diff
@@ -797,6 +809,10 @@ v8::Local<v8::Value> Window::From(v8::Isolate* isolate,
     return v8::Null(isolate);
 }
 
+void SetDeprecatedOptionsCheck(const DeprecatedOptionsCheckCallback& callback) {
+  g_deprecated_options_check = callback;
```

Comment:
@zcbenz I think we talked about this briefly but I just wanted to confirm that this call does or doesn't need the following after it: https://github.com/atom/electron/blob/67db92d500632548466149770e4ae2cd7445999b/atom/browser/api/atom_api_download_item.cc#L193-L194

```js
atom::AtomBrowserMainParts::Get()->RegisterDestructionCallback(
      base::Bind(DeprecatedOptionsCheckCallback));
```